### DISCUSSION TITLE
付款后没升级到专业版

### DISCUSSION THREAD

**Discussion by lunalina928** [getlantern/lantern-forum-cn#948] [General]
支付后，网页提示： ip [111.192.16.193] 不在白名单。 没升级成专业版
账号：763101129@qq.com 
![Screenshot_20241010_015310_Alipay](https://github.com/user-attachments/assets/36876592-998e-4b1a-9f89-39e827c70150)

**Comment by lunalina928** [comment]
@Mike777Khoo 看一下谢谢

**Comment by MaxMKMax** [comment]
@lunalina928 您好. 我已经帮您提交问题上去. 估计几天内会帮您处理好. 谢谢等待.
